lib/util/http/github.ts

```diff
@@ -210,31 +210,26 @@ export class GithubHttp extends Http<GithubHttpOptions, GithubHttpOptions> {
       if (opts.paginate) {
         // Check if result is paginated
         const pageLimit = opts.pageLimit ?? 10;
-        const linkHeader =
-          result?.headers?.link &&
-          parseLinkHeader(result.headers.link as string);
+        const linkHeader = is.string(result?.headers?.link)
+          ? parseLinkHeader(result.headers.link)
+          : null;
```

Comment:
```suggestion
        const linkHeader = parseLinkHeader(result?.headers?.link);
```
`parseLinkHeader` will handle undefined or null